cmd/evm/t8n_test.go

```diff
@@ -255,6 +255,10 @@ func TestT8n(t *testing.T) {
 }
 
 func TestEvmRun(t *testing.T) {
```

Comment:
Not sure what are requirements to meet for "short testing" but these tests are extremely simple and should be super fast. Maybe it's a good idea to measure deeper what makes it slow or maybe you know already?

Reply:
Fair enough. The threshold was 500ms. I've created follow-up Issue #15153